api/metrcis.go

```diff
@@ -75,6 +76,20 @@ type (
 		ContractID types.FileContractID
 		HostKey    types.PublicKey
 	}
+
+	WalletMetric struct {
+		Timestamp time.Time `json:"timestamp"`
+
+		Address types.Address `json:"address"`
```

Comment:
I think the address isn't very useful considering it never changes and is not something that you can visualise in a meaningful way.

Reply:
I was on the fence when I wrote it. I figured it couldn't hurt but removed it. It wasn't about ever needing to visualise multiple wallets, it was more to ensure we never visualise anything we know not be related to "our wallet".